app/components/MachinesPage/MachinesPage.tsx

```diff
@@ -20,10 +20,12 @@ export default function MachinesPage(props: {
 }): JSX.Element {
   // SET STATES
   const [submitSuccessful, setSubmitSuccessful] = useState(false);
+  const [buttonDisabled, setButtonDisabled] = useState(false);
```

Comment:
good call setting the state here so that the state setting function can be called with the event handler